docs/decisions/0001-course-experience.rst

```diff
@@ -0,0 +1,283 @@
+Courses units in LMS
+--------------
+
+Status
+======
+
+Proposed
+
+Context
+=======
+
+NELP business case requires multiples features that are not part
+of open-edx platform core, and it's necessary a new implementation
+that allows users to report and share their opinions about the
+course content and experience during the course.
+
+Decisions
+=========
+
+1. Create a course experience module.
+2. Create ``RateCourse`` model with the following fields.
+
+  * author => Foreign key to user model, the user who set their opinion.
+
+  * status => String field with choices: Liked, Unliked and not-set.
+
+  * course_id => String field, course identifier to allow course rating.
+
+3. Create ``RateSection`` model with the following fields.
+
+  * author => Foreign key to user model, the user who set their opinion.
+
+  * status => String field with choices: Liked, Unliked and not-set.
+
+  * usage_id => String field, this is course subsection identifier.
```

Comment:
why usage_id and not block_id?

Reply:
why do you think it's better block_id ?

Reply:
because the definition is using block_id. https://github.com/eduNEXT/eox-nelp/blame/5fe2d6c2436c29a1b27ceb4ac6b856ebdb19e1e1/docs/decisions/0001-course-experience.rst#L91
Also, I remember that the is a phrase like "everything is an xblock"

Reply:
after a small research I think a better option is `item_id`, because the current endpoint `/api/courseware/sequence/{usage_key}` returns this field, and that specific field  is currently used, in the microfrontend, to generate the iframe, so if we are lucky we can take advantage of that in the new react components